src/Identity/Core/src/SecurityStampValidator.cs

```diff
@@ -90,6 +90,13 @@ protected virtual async Task SecurityStampVerified(TUser user, CookieValidatePri
             // REVIEW: note we lost login authentication method
             context.ReplacePrincipal(newPrincipal);
             context.ShouldRenew = true;
+
+            if (!context.Options.SlidingExpiration)
```

Comment:
Should there be a null check on context.Options here? There's one in ValidateAsync below but I can't actually tell what that one is for.

Reply:
CookieValidatePrincipalContext's base constructor ensures Options is never null. Not sure why ValidateAsync checked it, it never even references Options otherwise. I'll remove it.
https://github.com/dotnet/aspnetcore/blob/e6a5af6633f08890afbd99a6ca7687c623454768/src/Security/Authentication/Core/src/Events/BaseContext.cs#L30-L33

Reply:
Ah, looks like leftovers from some refactoring during 2.0.